packages/route-viewer-overlay/src/RouteViewerOverlay.story.js

```diff
@@ -1,34 +1,76 @@
+/* eslint-disable react/jsx-props-no-spreading */
 import BaseMap from "@opentripplanner/base-map";
 import React from "react";
+import { GeoJSON } from "react-leaflet";
 
 import routeData from "../__mocks__/mock-route.json";
+import flexRouteData from "../__mocks__/mock-flex-route.json";
 import RouteViewerOverlay from ".";
 
 import "../../../node_modules/leaflet/dist/leaflet.css";
 
-const center = [45.543092, -122.671202];
+const PORTLAND = [45.543092, -122.671202];
+const POWDER_SPRINGS = [33.8595, -84.67483];
 const zoom = 11;
 
+// Hide some story args completely.
+const hiddenProp = {
+  table: { disable: true }
+};
+
 export default {
   title: "RouteViewerOverlay",
-  component: RouteViewerOverlay
+  component: RouteViewerOverlay,
+  argTypes: {
+    center: hiddenProp,
+    zoom: hiddenProp,
+    routeData: hiddenProp,
+    path: hiddenProp,
+    extraLayer: hiddenProp
+  }
 };
 
-export const Default = () => (
-  <BaseMap center={center} zoom={zoom}>
-    <RouteViewerOverlay routeData={routeData} visible />
+/* Unfortunately, args within an object can't be controlled via storybook controls.
+To be able to control the ClipToPatternStops arg via a control, the prop needs to be at 
+the root of the args object. This means that all props for both the BaseMap and the
+RouteViewerOverlay must be in the same args object. Luckily, there is no prop overlap
+and this is safely possible */
+const Template = args => (
+  <BaseMap {...args}>
+    <RouteViewerOverlay {...args} />
+    {args.extraLayer}
   </BaseMap>
 );
 
-export const WithPathStyling = () => (
-  <BaseMap center={center} zoom={zoom}>
-    <RouteViewerOverlay
-      path={{
-        opacity: 0.5,
-        weight: 10
-      }}
-      routeData={routeData}
-      visible
-    />
-  </BaseMap>
-);
+export const Default = Template.bind({});
+Default.args = {
+  center: PORTLAND,
+  zoom,
+  routeData
```

Comment:
Sort attributes (also in stories below).

Reply:
Sorted in f13469bc7ec8b24af48c993101ec3ca3bb69afcb